conf/mesos-site.xml

```diff
@@ -99,12 +109,12 @@
 
   <property>
     <name>mesos.hdfs.journalnode.cpus</name>
-    <value>1</value>
+    <value>0.5</value>
```

Comment:
Did we do any monitoring/benchmarking to arrive at this number, or is it just smaller so that it's easier to guarantee scheduling on a small/busy cluster?